cmake/tvm/pack_featgraph.py

```diff
@@ -0,0 +1,7 @@
+import os
+from sddmm_tr import sddmm_tree_reduction
+
+lib_path = '../../build/libfeatgraph.so'
```

Comment:
The path is hardcoded, we should create a function `find_lib_path` like https://github.com/dmlc/dgl/blob/ad8530a4dc2722e481f56c1d417c36cd87bbfb19/python/dgl/_ffi/libinfo.py#L8 to locate the libpath.